Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -18,11 +18,7 @@ First, install and/or load required packages and functions.
 knitr::opts_chunk$set(echo = TRUE)
 
 source('setup.R')
-library("readxl")
-library("trend")
```

Comment:
Just to verify, trend and viridis were not added to the setup script in case we do need them.

Reply:
I don't think they were actually used which is why I didn't add them.  Hope I'm not wrong.

Reply:
We'll find out soon enough LOL ;)